package.json

```diff
@@ -1,26 +1,62 @@
 {
-  "name": "hermez-landing",
+  "name": "hermez.io",
   "version": "1.0.0",
-  "description": "Landing page for the Hermez project",
+  "description": "Hermez Homepage",
   "author": "iden3",
   "scripts": {
-    "start": "live-server --port=3000 --mount=/:src --watch=src"
+    "build": "gatsby build",
+    "lint": "standard",
+    "lint:fix": "standard --fix",
+    "start": "gatsby develop -H 0.0.0.0",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean"
+  },
+  "dependencies": {
+    "clsx": "^1.1.1",
+    "gatsby": "^2.24.87",
+    "gatsby-plugin-react-svg": "^3.0.0",
+    "normalize.css": "^8.0.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-feather": "^2.0.8",
+    "react-helmet": "^6.1.0",
+    "react-jss": "^10.4.0",
+    "rss-parser": "^3.10.0"
```

Comment:
Why do we need the `rss-parser` dependency in this branch?

Reply:
I was experimenting with RSS feed, I'll remove it from here.